j2cl-maven-plugin/src/main/java/com/vertispan/j2cl/mojo/AnnotationProcessorMode.java

```diff
@@ -0,0 +1,22 @@
+package com.vertispan.j2cl.mojo;
+
+public enum AnnotationProcessorMode {
```

Comment:
document, esp with caveats, file a ticket for follow-up, and link the ticket here